setup.py

```diff
@@ -40,6 +40,7 @@
         'docs': ['sphinx_gallery', 'sphinx_rtd_theme', 'pydata_sphinx_theme', 'numpydoc',
                  'memory_profiler', 'pillow', 'ipython', 'sphinx_design', 'docstring_inheritance',
                  'lightning'],
+        'riemann': ['pyriemann', 'geoopt'],
```

Comment:
if it's just for an example I would put this in docs key no? it becomes required for the doc public.

also how do you test now that braindecode can work without geoopt?

Reply:
hmm, true @agramfort!

Reply:
solved @agramfort!